tests/dockerfile/Dockerfile

```diff
@@ -6,12 +6,13 @@ RUN dnf install -y git make gcc gcc-c++ which iproute iputils procps-ng vim-mini
 RUN npm install -g ts-node && npm install -g typescript
 ADD https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.10.10-bb74230f.tar.gz /geth/
 RUN cd /geth && tar -xvf * && mv /geth/**/geth /usr/bin/geth
-# Download gaia as a IBC test chain
-ADD https://github.com/cosmos/gaia/releases/download/v7.0.0/gaiad-v7.0.0-linux-amd64 /usr/bin/gaiad
-# Setup Hermes for IBC connections between chains
-ADD https://github.com/informalsystems/ibc-rs/releases/download/v0.13.0/hermes-v0.13.0-x86_64-unknown-linux-gnu.tar.gz /tmp/
-RUN cd /tmp/ && tar -xvf hermes-v0.13.0-x86_64-unknown-linux-gnu.tar.gz && mv hermes /usr/bin/
-RUN mkdir /ibc-relayer-logs && touch /ibc-relayer-logs/hermes-logs && touch /ibc-relayer-logs/channel-creation
+# Download modified gaia as a IBC test chain, IMO should be replaced with git clone or something
+ADD https://github.com/althea-net/gaia-ica/releases/download/v7.0.3/gaiad-v7.0.3-linux-amd64 /usr/bin/gaiad
```

Comment:
This is the link to the althea-net repo I created to mimic the work gadost did. Gadost had a separate copy of the icaauth module that this branch of Gravity now contains, I simplified the changes by simply importing this branch of Gravity into althea-net/gaia-ica. Once this branch gets merged I will need to clean up that repo to point to main, or some particular version of Gravity.